app/services/katello/pulp3/content_view_version.rb

```diff
@@ -15,10 +16,14 @@ def generate_exporter_id
       end
 
       def generate_exporter_path
-        export_path = "#{@content_view_version.content_view}/#{@content_view_version.version}".gsub(/\s/, '_')
+        export_path = "#{@content_view_version.content_view}/#{@content_view_version.version}/#{@destination_server}/#{increment_path}".gsub(/\s/, '_')
```

Comment:
would a datestamp be more meaningful than an incrementing number?    2020-02-01-12-34?